packages/itwin/tree-widget/src/tree-widget-react/components/trees/common/internal/Utils.ts

```diff
@@ -108,3 +108,8 @@ export function getClassesByView(viewType: "2d" | "3d") {
     ? ({ categoryClass: CLASS_NAME_DrawingCategory, elementClass: CLASS_NAME_GeometricElement2d, modelClass: CLASS_NAME_GeometricModel2d } as const)
     : ({ categoryClass: CLASS_NAME_SpatialCategory, elementClass: CLASS_NAME_GeometricElement3d, modelClass: CLASS_NAME_GeometricModel3d } as const);
 }
+
+/** @internal */
+export function getArrayFromId64Arg(arg: Id64Arg): Id64Array {
```

Comment:
As far as I can tell this function is used to create an array, which we later, in all cases, convert to a string. Would it make more sense to instead have `reduceId64Arg`? It would allow us to skip possibly unnecessary conversion from Set to Array.